.github/workflows/build.yml

```diff
@@ -27,6 +27,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
+      - name: Create .env file
```

Comment:
Its just a proposal: 

But maybe this way we can fix e2e-Tests finally

Reply:
I think as part of the `npm install` we can use the `postinstall` hook to copy the `.env.template` file to `.env`, that way it will work for people locally too

Reply:
I didnt think about that. This is actually much better than using a custo action for this. 
I'll revert this and add a `postinstall hook to package.json`.

Reply:
Good point 👍 I would use a npm package to do the copy to solve this issue, something like https://www.npmjs.com/package/copyfiles